app/controllers/api/account.php

```diff
@@ -682,7 +682,7 @@
                 'emailVerification' => false,
                 'status' => Auth::USER_STATUS_UNACTIVATED,
                 'password' => null,
-                'passwordUpdate' => \time(),
+                'passwordUpdate' => 0,
```

Comment:
I don't think we should allow anonymous users to change their password. Since they don't have an E-Mail address yet.

When Updating the E-Mail address, the user needs to pass an E-Mail & their current Password. In the case of an anonymous user, the passed password will become the new password and the account is converted to a normal account.

Reply:
Okay makes sense 👍